content/applications/general/iot/windows_iot.rst

```diff
@@ -0,0 +1,158 @@
+===================
+Windows virtual IoT
+===================
+
+To start using the Windows virtual IoT:
+
+#. Make sure all :ref:`prerequisites <iot/windows-iot/prerequisites>` are met.
+#. :ref:`Install the Windows virtual IoT <iot/iot_box/network>` on a Windows computer.
+#. :ref:`Configure the Windows firewall <iot/windows-iot/firewall>`.
+#. Connect your :doc:`devices <devices>` to the Windows virtual IoT.
+#. :doc:`Connect the Windows virtual IoT to your Odoo database <connect>`.
+
+.. _iot/windows-iot/prerequisites:
+
+Prerequisites
+=============
+
+The following prerequisites must be met before setting up and using the Windows virtual IoT:
+
+- A valid IoT box subscription.
+- An updated and recent version of Windows (i.e., Windows 10 or Windows 11) installed on a Windows
+  computer (laptop, desktop, or server).
+
+.. note::
+   - :abbr:`MRP (Material Requirement Planning)` devices, including cameras and measurement tools,
+     are not compatible with Windows virtual IoT.
+   - It is also possible to create a Windows Virtual Machine on a MacOS/Linux computer. However,
+     this option is not supported by Odoo, and no troubleshooting assistance will be provided.
+
+.. _iot/windows-iot/installation:
+
+Installation
+============
+
+To install the Windows virtual IoT on a Windows computer:
+
+#. Access `Odoo's download page <https://odoo.com/download>`_ and download the latest Odoo
+   installation package for Windows.
+#. Open the downloaded :file:`.exe` file, allow the app to make changes to your device, select
+   a language, and click :guilabel:`OK`.
+#. Click :guilabel:`Next`, then :guilabel:`I Agree` to accept the terms and conditions and continue.
+#. Select :guilabel:`Odoo IoT` from the :guilabel:`Select the type of install` dropdown list. The
+   following components should be selected: Odoo Server, Odoo IoT, Nginx WebServer, and Ghostscript
+   interpreter.
+#. Verify you have the required space on your computer and click :guilabel:`Next`.
+#. In the :guilabel:`Destination folder`, enter `C:\odoo` and click :guilabel:`Install`.
```

Comment:
the `\` does not appear in the doc
![image](https://github.com/user-attachments/assets/1da34373-02d1-49b0-96f3-4b7a97434033)